snapcraft/cli/_errors.py

```diff
@@ -86,18 +96,26 @@ def exception_handler(  # noqa: C901
         - a snapcraft handled error occurs, debug=False so only the
           exception message is shown
     """
-    # TODO pickle the traceback if on a manged host so the actual host can deal with it.
     exc_info = (exception_type, exception, exception_traceback)
     exit_code = 1
-    is_snapcraft_error = issubclass(exception_type, errors.SnapcraftError)
+    is_snapcraft_error = (
```

Comment:
mind adding a comment above explaining why we filter out ProviderExecError for the casual future code contributor (including ourselves :smile:)?

Reply:
Added. It's better to do that while things still make sense.